web/packages/teleport/src/Discover/SelectResource/databases.tsx

```diff
@@ -421,10 +421,27 @@ export function getDatabaseProtocol(engine: DatabaseEngine): DbProtocol {
   switch (engine) {
     case DatabaseEngine.Postgres:
     case DatabaseEngine.AuroraPostgres:
+    case DatabaseEngine.Redshift:
       return 'postgres';
     case DatabaseEngine.MySql:
     case DatabaseEngine.AuroraMysql:
       return 'mysql';
+    case DatabaseEngine.MongoDb:
+      return 'mongodb';
+    case DatabaseEngine.Redis:
+      return 'redis';
+    case DatabaseEngine.CockroachDb:
+      return 'cockroachdb';
+    case DatabaseEngine.SqlServer:
+      return 'sqlserver';
+    case DatabaseEngine.Snowflake:
+      return 'snowflake';
+    case DatabaseEngine.Cassandra:
+      return 'cassandra';
+    case DatabaseEngine.ElasticSearch:
+      return 'elasticsearch';
+    case DatabaseEngine.DynamoDb:
+      return 'dynamodb';
```

Comment:
We can make this switch exhaustive by adding:
```ts
    default:
      engine satisfies never;
```
at the end. This will return a TS error when someone adds a new DB engine but not provide a protocol for it here.
We can also remove `return '' as any`.

Reply:
thank you! This actually caught that I missed the `doc` engine type and highlighted it in my editor. However it doesn't seem to cause any build failure when I rebuild Teleport.